media/starboard/progressive/avc_parser.cc

```diff
@@ -27,7 +27,8 @@
 #include "media/base/video_color_space.h"
 #include "media/base/video_transformation.h"
 #include "media/base/video_types.h"
-#include "media/formats/mp4/aac.h"
+// TODO(b/377295011): remove the nogncheck annotation.
```

Comment:
Let's replace this and below TODOs with `TODO(cobalt, b/123)`

Reply:
Done